tests/test_backup_restore.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from playwright.sync_api import Page, expect
+from test_user_update_and_app_install import login_user
+from test_helpers import run_make_command, assert_backup_files_exist, wait_for_service_healthy
+
+
+@pytest.mark.order(1)
+def test_launch_environment():
+    run_make_command("launch COMPOSE_OPTS=-d")
+
+
+@pytest.mark.order(3)
+def test_create_backup():
+    backup_timestamp = pytest.backup_timestamp
+
+    run_make_command("backup", {"BACKUP_TIMESTAMP": backup_timestamp})
+    assert_backup_files_exist(backup_timestamp)
+
+
+@pytest.mark.order(4)
+def test_clean_environment():
+    run_make_command("clean")
+
+
+@pytest.mark.order(5)
+def test_launch_fresh_environment():
+    run_make_command("launch COMPOSE_OPTS=-d")
```

Comment:
Assert services are running as expected?

Reply:
Do we really need more than the compose healthchecks?